src/math/Color.js

```diff
@@ -334,23 +347,33 @@ class Color {
 
 	}
 
-	getHex() {
+	getHex( colorSpace = sRGBColorSpace ) {
```

Comment:
We'll need a method to get the r/g/b color components out in an explicit color space. The model exporters, for example, often write the r/g/b components out explicitly in a specific color space. I propose something like `Color.getRGB( target, colorSpace )` where "target" is a basic object with r, g, b fields.

Reply:
Sounds good to me!

Reply:
Instead, I would follow the existing pattern:
```js
color.convertSRGBToLinear();

color.copySRGBToLinear( source_color );
```

The operation is applied to `this`. The inputs are not changed.

Reply:
For the use case @gkjohnson is describing (like an reusable exporter, or perhaps retrieving a color to display in a color picker) the advantage of providing a target color space is that it's more robust against different contexts:

1. color management might be disabled – see 3b and 3c above; the value is already sRGB
2. working color space might (in the future) be something other than linear-srgb, like ACEScg

With an API like `color.getRGB( target, sRGBColorSpace )` an exporter can reliably get a color in the expected color space, regardless of other application state.

Reply:
My point is, it should be `color.convert( source_color, fromColorSpace, toColorSpace );` The args are not changed; "this" is.

Also, how do you convert from one color space to another with out specifying both color spaces?

Reply:
^This is how I've been thinking about it as well – if we agree, a `Color` instance is always in the working color space, and only does a conversion to other color spaces when serialized. The main flies in that ointment are `scene.fog` and `scene.background`, which should _not_ be given in the working color space at the moment. But those would change if we do the output transforms in a post-processing pass rather than the shader.